### PR TITLE
feat(sign-up): added asterisks to indicate mandatory fields

### PR DESCRIPTION
**Pull request by oalkabouss**
ref: MANAGER-13345

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13345
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/sign-up/src/form/sign-up.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.required-field {
+  .oui-field__label::after {
+    content: ' *';
+    color: #c11b1b;
```
Comment (JacquesLarique) [inline packages/manager/modules/sign-up/src/form/sign-up.scss:4]: Can you use css variables please ?
Here you can use `$ae-500`
And for the font-weight below, you could use `$jupiter-font-weight`

**Review by JacquesLarique — CHANGES_REQUESTED** [review]
I feel like it would be easier to read using:
`ng-class="{ 'required-field' : ctrl.signUpFormCtrl.rules[<field>].mandatory}]"`

rather than
`class="{{$ctrl.signUpFormCtrl.rules[<field>].mandatory ? 'required-field' : ''}}"`

File: packages/manager/apps/sign-up/src/form/form.html [hunk 2]
```diff
@@ -75,4 +75,8 @@
             </oui-step-form>
         </oui-stepper>
     </ovh-sign-up-form>
+    <div data-ng-if="!$ctrl.isIndiaSubsidiary" class="mt-2">
+        <span class="mandatory-fields-star">*</span>
+        <i data-translate="sign_up_form_mandatory_fields"></i>
+    </div>
```
Comment (JacquesLarique) [inline packages/manager/apps/sign-up/src/form/form.html:81]: Why don't we display this to Indian customer ?
I don't see any request for this behavior in the ticket

Comment (oalkabouss) [reply packages/manager/apps/sign-up/src/form/form.html]: Thanks, I forgot to delete it!

**Review by anooparveti — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Pull request merged**